package.json

```diff
@@ -11,15 +11,17 @@
     "dev": "ts-node-dev --respawn --transpileOnly src/server.ts",
     "test": "jest",
     "tsc": "tsc --noEmit",
-    "build": "tsc",
-    "prebuild": "rm -rf target",
-    "postbuild": "cp package.json target/ && cp yarn.lock target/ && cd target && yarn install --production && cd .. && ./bin/bundle.sh",
     "tslint": "tslint -c tslint.json 'src/**/*.ts'",
     "fix": "tslint -c tslint.json --fix 'src/**/*.ts'",
     "cdk": "cdk",
-    "deploy": "cdk --profile frontend deploy",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "synth": "cdk --profile frontend synth > cloudformation.yaml",
+    "deploy": "cdk --profile frontend deploy",
+    "prebuild": "rm -rf target",
+    "build": "tsc",
+    "postbuild": "cp package.json target/ && cp yarn.lock target/ && cd target && yarn install --production && cd .. && cdk --profile frontend synth > cloudformation.yaml",
+    "upload-artefact": "node-riffraff-artifact"
```

Comment:
Minor one here but still worth mentioning - everywhere else we use `artifact` so it's probably better to rename this script to use the same terminology (as opposed to `artefact`) - good to rely on the same terminology.

Reply:
ha yes this was somewhat a joke - see https://github.com/guardian/node-riffraff-artifact and https://github.com/guardian/node-riffraff-artefact. Having said that, the Guardian style guide recommends the 'e' version! https://www.theguardian.com/guardian-observer-style-guide-a. I'll change though as unfortunately I think you're right that consistency is more important.